src/NBitcoin/BitcoinStream.cs

```diff
@@ -379,15 +379,16 @@ public IDisposable ProtocolVersionScope(ProtocolVersion version)
             });
         }
 
-        public void CopyParameters(BitcoinStream stream)
-        {
-            if(stream == null)
-                throw new ArgumentNullException("stream");
-            ProtocolVersion = stream.ProtocolVersion;
-            IsBigEndian = stream.IsBigEndian;
-            MaxArraySize = stream.MaxArraySize;
-            Type = stream.Type;
-        }
+		public void CopyParameters(BitcoinStream stream)
+		{
+			if(stream == null)
+				throw new ArgumentNullException("stream");
+			ProtocolVersion = stream.ProtocolVersion;
+            TransactionOptions = stream.TransactionOptions.Clone();
```

Comment:
You're reintroducing tabs. Please use spaces.
That goes to every file in this PR- untabify it back.

Reply:
Done. Possibly merge issues carrying over from older versions. My editor is set correctly.